src/main/groovy/org/mockito/release/internal/gradle/GitCommitTask.java

```diff
@@ -0,0 +1,39 @@
+package org.mockito.release.internal.gradle;
+
+import org.gradle.api.Task;
+import org.gradle.api.tasks.Exec;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+public class GitCommitTask extends Exec{
+
+    private List<File> filesToCommit = new ArrayList<File>();
+    private List<String> messages = new ArrayList<String>();
+
+    public void addChange(List<File> files, String message, Task taskMakingChange){
```

Comment:
Nitpick: let's call it 'description' as a description of the change. 'message' might be confused with commit message which is something different, something constructed from descriptions.